simulation/simControl.go

```diff
@@ -1345,7 +1345,6 @@ func StartSimControl(w *worker.Thread, listenTo int, listenStdin bool) {
 				case 'L' == b[0]:
 					if len(b) <= 2 {
 						if cancelheight == -1 {
-							fmt.Errorf("Exp LH.I")
```

Comment:
Change these to print the error out instead? The rest of the file seems to use either `fmt.Println` or `os.StdErr.WriteString`

Reply:
Changed for Println